src/app/components/RadioSchedule/ProgramCard/index.jsx

```diff
@@ -128,6 +129,11 @@ ProgramCard.propTypes = {
     link: string.isRequired,
     brandTitle: string.isRequired,
   }).isRequired,
+  id: string,
+};
+
+ProgramCard.defaultProps = {
+  id: '1',
```

Comment:
Is there ever a scenario where `id` is not passed to this component?

If there is, I think there will be issues in the ScheduleItemHeader component, because it will then emit multiple elements with the same ID (`scheduleItem-1`)

If ID is always passed, I think this prop can just be marked as `.isRequired` above and the default deleted :)

Reply:
I set it with a default as there is a snapshot test that uses an undefined ID. Might be better to pass in a test ID for it and change to isRequired :)